impl/src/rendering/bazel.rs

```diff
@@ -275,18 +294,36 @@ impl BuildRenderer for BazelRenderer {
       })
     }
 
-    let alias_file_path = path_prefix.as_path().join(buildfile_suffix);
-    let rendered_alias_build_file = self
-      .render_remote_aliases(&workspace_context, &crate_contexts)
-      .map_err(|e| RazeError::Rendering {
-        crate_name_opt: None,
-        message: unwind_tera_error!(e),
-      })?;
+    for member_path in planned_build.workspace_context.workspace_members.iter() {
+      let all_packages: Vec<CrateContext> = planned_build
```

Comment:
This block seems to duplicate the code above.

Reply:
Can you elaborate?

Reply:
L222
```
    for member_path in planned_build.workspace_context.workspace_members.iter() {
      let all_packages: Vec<CrateContext> = planned_build
        .crate_contexts
        .iter()
        .filter(|ctx| {
          ctx.is_binary_dependency || ctx.workspace_member_dependents.contains(member_path)
        })
        .map(|ctx| ctx.clone())
        .collect();
      let rendered_alias_build_file = self
        .render_aliases(&workspace_context, &all_packages)
        .map_err(|e| RazeError::Rendering {
          crate_name_opt: None,
          message: unwind_tera_error!(e),
        })?;

      file_outputs.push(FileOutputs {
        path: render_details
          .cargo_root
          .join(member_path)
          .join(&render_details.workspace_member_output_dir)
          .join("BUILD.bazel"),
        contents: rendered_alias_build_file,
      });
    }
```

L292
```
      let all_packages: Vec<CrateContext> = planned_build
        .crate_contexts
        .iter()
        .filter_map(|ctx| {
          if ctx.is_binary_dependency {
            Some(ctx.clone())
          } else if ctx.workspace_member_dependents.contains(member_path) {
            Some(ctx.clone())
          } else {
            None
          }
        })
        .collect();
      let rendered_alias_build_file = self
        .render_remote_aliases(&workspace_context, &all_packages)
        .map_err(|e| RazeError::Rendering {
          crate_name_opt: None,
          message: unwind_tera_error!(e),
        })?;

      file_outputs.push(FileOutputs {
        path: render_details
          .cargo_root
          .join(member_path)
          .join(&render_details.workspace_member_output_dir)
          .join("BUILD.bazel"),
        contents: rendered_alias_build_file,
      });
```